Makefile

```diff
@@ -510,7 +510,7 @@ ifeq ($(shell test $(KDF_ITERATIONS) -gt 0; echo $$?),0)
   NIM_PARAMS += -d:KDF_ITERATIONS:"$(KDF_ITERATIONS)"
 endif
 
-NIM_PARAMS += -d:chronicles_sinks=textlines[stdout],textlines[nocolors,dynamic],textlines[file,nocolors] -d:chronicles_runtime_filtering=on -d:chronicles_default_output_device=dynamic
+NIM_PARAMS += -d:chronicles_sinks=textlines[stdout],textlines[nocolors,dynamic],textlines[file,nocolors] -d:chronicles_runtime_filtering=on -d:chronicles_default_output_device=dynamic -d:chronicles_log_level=trace
```

Comment:
Wouldn't this set the _default_ log level to `trace`? :thinking:

Reply:
> Wouldn't this set the default log level to trace?

And yes and no. 🙂

Yes, that sets the log level, but for the `nim-chronicles`, not for the Status app.

The default log level for the Status app is set here in [this line](https://github.com/status-im/status-desktop/blob/c5d1d5b77d881e25393ac19aab64e88d4943d614/src/env_cli_vars.nim#L10).

The goal is that `nim-chronicles` can accept any log level set by the Status app, which is now possible.

The reason why it was not possible earlier is that the default `nim-chronicles`'s log level (`INFO` in case of release, `DEBUG` otherwise) was higher than the log level that can be set by the app (`TRACE`).